pkg/controller/disruption/disruption_test.go

```diff
@@ -492,21 +396,32 @@ func TestBuildDisruptedMachineMap(t *testing.T) {
 }
 
 func TestCountHealthyMachines(t *testing.T) {
-	healthyNode := newNode("healthyNode", true)
-	unhealthyNode := newNode("unhealthyNode", false)
+	// healthy machine
+	healthyNode := maotesting.NewNode("healthyNode", true)
+	healthyMachine := maotesting.NewMachine("healthyMachine", healthyNode.Name)
+
+	// unhealthy machine
+	unhealthyNode := maotesting.NewNode("unhealthyNode", false)
+	unhealthyMachine := maotesting.NewMachine("unhealthyMachine", unhealthyNode.Name)
 
 	currentTime := metav1.NewTime(time.Now())
 	timeAfterThreeMinutes := metav1.NewTime(currentTime.Add(3 * time.Minute))
 	timeBeforeThreeMinutes := metav1.NewTime(currentTime.Add(-3 * time.Minute))
 
-	healthyMachine := newMachine("healthyMachine", healthyNode.Name, "Running")
-	unhealthyMachine := newMachine("unhealthyMachine", unhealthyNode.Name, "Running")
-	deletedMachine := newMachine("deletedMachine", healthyNode.Name, "Running")
+	// deleted machine
+	deletedNode := maotesting.NewNode("deletedNode", true)
+	deletedMachine := maotesting.NewMachine("deletedMachine", deletedNode.Name)
 	deletedMachine.DeletionTimestamp = &currentTime
-	disruptedMachineBeforeTimeout := newMachine("disruptedMachineBeforeTimeout", healthyNode.Name, "Running")
-	disruptedMachineAfterTimeout := newMachine("disruptedMachineAfterTimeout", healthyNode.Name, "Running")
 
-	r := newFakeReconciler(nil, healthyNode, unhealthyNode)
+	// machine disrupted before the timeout
+	nodeDisruptedBeforeTimeout := maotesting.NewNode("nodeDisruptedBeforeTimeout", true)
+	disruptedMachineBeforeTimeout := maotesting.NewMachine("disruptedMachineBeforeTimeout", nodeDisruptedBeforeTimeout.Name)
+
+	// machine disrupted after the timeout
+	nodeDisruptedAfterTimeout := maotesting.NewNode("nodeDisruptedAfterTimeout", true)
+	disruptedMachineAfterTimeout := maotesting.NewMachine("disruptedMachineAfterTimeout", nodeDisruptedAfterTimeout.Name)
+
+	r := newFakeReconciler(nil, healthyNode, unhealthyNode, deletedNode, nodeDisruptedBeforeTimeout, nodeDisruptedAfterTimeout)
```

Comment:
Does not look like just moving the code from from place to another. Before the change `newFakeReconciler` accepted three arguments. Now, it accepts 6. Please, separate this change into another commit for better readability.

Reply:
done